Feature/mv unused dev

XianwuXue-NOAA opened the pull request.

XianwuXue-NOAA committed:
1) Improve all build_*.sh to print out all message, especially for "module purge"
2) Clean up some build_g*.sh to remove unused statements

 On branch feature/mv_unused_dev
 Changes to be committed:
	modified:   build_all.sh
	modified:   build_gefs_anom2_fcst.sh
	modified:   build_gefs_ensstat.sh
	modified:   build_gefs_nstgen.sh
	modified:   build_global-workflow.sh
	modified:   build_global_ensadd.sh
	modified:   build_global_ensppf.sh
	modified:   build_wave_stat.sh

Refs: #76266, #64583, #62644, #71500 and #56936

XianwuXue-NOAA committed:
Make the default value of "fhrestart" from 48 to 24

 On branch feature/mv_unused_dev
 Changes to be committed:
	modified:   ../parm/gefs.parm
	modified:   user_full.conf

Refs: #76266, #64583, #62644, #71500 and #56936

XianwuXue-NOAA committed:
Merge remote-tracking branch 'upstream/master' into feature/mv_unused_dev

XianwuXue-NOAA committed:
Change the default value of "RocotoGen" from 1 to 0 to use variables in rocoto/param

 On branch feature/mv_unused_dev
	modified:   bin/wcoss_dell_p35/common.sh

Refs: #76266, #64583, #62644, #71500 and #56936

WalterKolczynski-NOAA commented on sorc/build_gefs_ensstat.sh:
```diff
@@ -1,22 +1,21 @@
 set -eux
 
-source ./machine-setup.sh > /dev/null 2>&1
+source ./machine-setup.sh #> /dev/null 2>&1
```
Remove these tail comments if this is the way it is going to stay

XianwuXue-NOAA replied:
OK. I will remove them today after the ecflow meeting. Also I want to modify the versions today.

XianwuXue-NOAA replied:
Just removed all tail comments and update the version/gefs.ver.

WalterKolczynski-NOAA reviewed (changes requested):
Remove the tail comments where you are no longer redirecting to /dev/null

XianwuXue-NOAA committed:
1) Cleanup the build_XX.sh
2) Improve versions/gefs.ver using versions/gefs_wcoss_dell_p35.ver

 On branch feature/mv_unused_dev
 Changes to be committed:
	modified:   sorc/build_gefs_anom2_fcst.sh
	modified:   sorc/build_gefs_ensstat.sh
	modified:   sorc/build_gefs_nstgen.sh
	modified:   sorc/build_global_ensadd.sh
	modified:   sorc/build_global_ensppf.sh
	modified:   sorc/build_wave_stat.sh
	modified:   versions/gefs.ver

Refs: #76266, #64583, #62644, #71500 and #56936

XianwuXue-NOAA committed:
Moidfy the value of gefs_ver to v12.0.0.0

 On branch feature/mv_unused_dev
 Changes to be committed:
	modified:   versions/gefs.ver
	modified:   versions/gefs_wcoss_dell_p3.ver
	modified:   versions/gefs_wcoss_dell_p35.ver

Refs: #76266, #64583, #62644, #71500 and #56936

XianwuXue-NOAA commented:
I just submitted the last commit. Please go ahead to merge it if no objections.

WalterKolczynski-NOAA commented on versions/gefs.ver:
```diff
@@ -1,8 +1,26 @@
-export util_shared_ver=v1.0.5
-#export enkf_ver=`cat $NWROOTp1/versions/gfs.ver |grep enkf_ver |awk -F"=" '{print $2}'`
-#export post_ver=`cat $NWROOTp1/versions/gfs.ver |grep post_ver |awk -F"=" '{print $2}'`
+export gefs_ver=v12.0.0.0
```
This is one too many version levels. GEFS should be v12.0.0

XianwuXue-NOAA replied:
Dingchen said "NCO will finalize the 3 section name. A fourth section give us more freedom before NCO finalization"

WalterKolczynski-NOAA replied:
Properly, semantic version numbers should be v12.0.0-0 or v12.0.0-beta if that is what he is going for https://semver.org/

XianwuXue-NOAA replied:
I will send an email to Dingchen and you to discuss the gefs_ver.
